frontend/src/routes/(authenticated)/project/[project_code]/BulkAddProjectMembers.svelte

```diff
@@ -34,9 +34,10 @@
   $: addedCount = addedMembers.length + createdMembers.length;
 
   const usernameRe = /^[a-zA-Z0-9_]+$/;
+  const emailRe = /@/;
 
-  function validateUsernames(usernames: string[]): boolean {
-    return usernames.every(s => usernameRe.test(s));
+  function validateBulkAddInput(usernames: string[]): boolean {
+    return usernames.every(s => usernameRe.test(s) || emailRe.test(s));
```

Comment:
It would be a bit more satisfying to use `zod`'s email validation.
I just opened [a PR](https://github.com/sillsdev/languageforge-lexbox/pull/733/files#diff-cb79a74b4cdfdc098413d7ba353d2ce7c8da7268d5afde368b0155cc6f0dd502R15) that adds a util for doing that like so:
```suggestion
    return usernames.every(s => usernameRe.test(s) || isEmail(s));
```